### PR TITLE
Add image prefix for new functions

### PR DESCRIPTION
**Pull request by iyovcheva**
Signed-off-by: Ivana Yovcheva <iyovcheva@vmware.com>

### CODE REVIEW

**Comment by alexellis** [comment]
Sorry we ended up with some conflicting files due to another merged PR. Can you rebase in the morning please?

**Comment by alexellis** [comment]
Strangely I can't see you on this list either, but I am sure we merged at least one of your PRs already - https://github.com/openfaas/workshop/graphs/contributors

**Comment by iyovcheva** [comment]
@alexellis the conflicts are resolved now

File: lab3.md [hunk 1]
```diff
@@ -80,8 +80,6 @@ functions:
     image: hello-openfaas
 ```
 
-On the line `image: ` make sure the name is prefixed with your Docker Hub account. For Alex Ellis this is `image: alexellis2/hello-openfaas`. Update this for your name every time you create a new function.
```
Comment (alexellis) [inline lab3.md]: Can we re-introduce a sentence like this and explain the `--prefix` flag at the same time?

**Pull request merged**